core/server/worker/src/main/java/alluxio/worker/block/TieredBlockStore.java

```diff
@@ -85,8 +88,14 @@
 public class TieredBlockStore implements BlockStore {
   private static final Logger LOG = LoggerFactory.getLogger(TieredBlockStore.class);
 
+  private static final boolean RESERVER_ENABLED =
+      Configuration.getBoolean(PropertyKey.WORKER_TIERED_STORE_RESERVER_ENABLED);
+  private static final long FREE_SPACE_TIMEOUT =
```

Comment:
Is this in MS? Also, why is this timeout a `long` and the one below is an `int`?

Reply:
Updated, the other one is an int due to the signature of the retry policies